mage_ai/api/policies/OauthPolicy.py

```diff
@@ -23,6 +72,7 @@ class OauthPolicy(BasePolicy):
 OauthPolicy.allow_actions(
     [
         constants.DETAIL,
+        constants.LIST,
```

Comment:
Question: Is it safe to return the oauth urls publicly? Are there any sensitive data in the response?

Reply:
no there shouldn't be any sensitive data in the initial oauth redirect url